src/view/configuration/configuration.jsx

```diff
@@ -110,6 +112,46 @@ const getInitialInstanceValues = getMergedBridgeMethod(
 );
 const getInstanceSettings = getMergedBridgeMethod("getInstanceSettings");
 
+/**
+ * Creates the validation schema based on library type
+ */
+const createValidationSchema = () => {
```

Comment:
Dynamically evaluates the validation schema based on the `libraryCode.type` state